core/src/consensus/mod.rs

```diff
@@ -54,16 +54,7 @@ use crate::scheme::CommonParams;
 use crate::views::HeaderView;
 use Client;
 
-/// Seal type.
-#[derive(Debug, PartialEq, Eq)]
-pub enum Seal {
-    /// Proposal seal; should be broadcasted, but not inserted into blockchain.
-    Proposal(Vec<Bytes>),
-    /// Regular block seal; should be part of the blockchain.
-    Regular(Vec<Bytes>),
-    /// Engine does generate seal for this block right now.
-    None,
-}
+pub type Seal = Option<Vec<Bytes>>;
```

Comment:
What is the available format of the Seal?

Reply:
How about create an Enum has three types?

Reply:
It's a good idea. I'll start right now.